components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -1649,36 +1630,28 @@ void JsonRpcService::EnsGetEthAddr(const std::string& domain,
     return;
   }
 
-  if (EnsL2FeatureEnabled()) {
-    if (ens_get_eth_addr_tasks_.ContainsTaskForDomain(domain)) {
-      ens_get_eth_addr_tasks_.AddCallbackForDomain(domain, std::move(callback));
-      return;
-    }
-
-    absl::optional<bool> allow_offchain;
-    if (EnsOffchainPrefEnabled(local_state_prefs_)) {
-      allow_offchain = true;
-    } else if (EnsOffchainPrefDisabled(local_state_prefs_)) {
-      allow_offchain = false;
-    }
-
-    // JsonRpcService owns EnsResolverTask instance, so Unretained is safe here.
-    auto done_callback = base::BindOnce(
-        &JsonRpcService::OnEnsGetEthAddrTaskDone, base::Unretained(this));
-
-    ens_get_eth_addr_tasks_.AddTask(
-        std::make_unique<EnsResolverTask>(
-            std::move(done_callback), api_request_helper_.get(),
-            api_request_helper_ens_offchain_.get(), MakeAddrCall(domain),
-            domain, GetEnsRpcUrl(), allow_offchain),
-        std::move(callback));
+  if (ens_get_eth_addr_tasks_.ContainsTaskForDomain(domain)) {
+    ens_get_eth_addr_tasks_.AddCallbackForDomain(domain, std::move(callback));
     return;
   }
 
-  auto internal_callback = base::BindOnce(
-      &JsonRpcService::ContinueEnsGetEthAddr, weak_ptr_factory_.GetWeakPtr(),
-      domain, std::move(callback));
-  EnsRegistryGetResolver(domain, std::move(internal_callback));
+  absl::optional<bool> allow_offchain;
+  if (EnsOffchainPrefEnabled(local_state_prefs_)) {
+    allow_offchain = true;
+  } else if (EnsOffchainPrefDisabled(local_state_prefs_)) {
+    allow_offchain = false;
+  }
+
+  // JsonRpcService owns EnsResolverTask instance, so Unretained is safe here.
+  auto done_callback = base::BindOnce(&JsonRpcService::OnEnsGetEthAddrTaskDone,
+                                      base::Unretained(this));
+
+  ens_get_eth_addr_tasks_.AddTask(
+      std::make_unique<EnsResolverTask>(
+          std::move(done_callback), api_request_helper_.get(),
+          api_request_helper_ens_offchain_.get(), MakeAddrCall(domain), domain,
+          GetEnsRpcUrl(), allow_offchain),
+      std::move(callback));
 }
 
```

Comment:
`EnsRegistryGetResolver`` no longer referenced

Reply:
https://github.com/brave/brave-core/pull/20898/commits/489e3989807691bf686cbe89e390e9b14e9cd5cb

Reply:
More removed in https://github.com/brave/brave-core/pull/20898/commits/54020b87056b1d70d2f165524b867b753be732b3.